dev/com.ibm.websphere.appserver.features/visibility/public/messaging-3.0/resources/l10n/io.openliberty.messaging-3.0.properties

```diff
@@ -0,0 +1,20 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the configuration of resource adapters to access messaging systems using \
+ the Jakarta Message Service API. This also includes the configuration of Jakarta Message Service connection \
```

Comment:
`This also includes the configuration of Jakarta Message Service connection \
 factories, queues, topics and activation specifications.`

What does "This" refer to ? The feature?

Reply:
"This" is the feature, messaging-3.0.  The verb is now changed to enables rather than includes.